dev-scripts/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["../.eslintrc.json"],
+  "rules": {
```

Comment:
I don't think these rules are needed if they're defined in the base. This should be enough (?)

```json
{
  "extends": ["../.eslintrc.json"]
}
```

Reply:
Nevermind, I missed the difference in `allow`. Maybe just allow them all in the base?

Reply:
Nah, I want to be strict about what can be used in client (ideally no console).

I can even forbid the console group, and add inline eslint comment if you prefer that.

Reply:
I'm fine with it then 👍🏻